nimbis-ppc-gen-password

```diff
@@ -29,7 +29,6 @@ pwd_context = LazyCryptContext(
 
     # set some useful global options
     default="sha256_crypt" if sys_bits < 64 else "sha512_crypt",
-    all__vary_rounds = 0.1,
```

Comment:
Looks good, Alex.

My preference for commit messages is to _always_ have something in addition the single line summary. For a commit like this, I would recommend adding something like:

As of passlib 1.7 we were getting a deprecation warning such as:

passlib/utils/handlers.py:1684: PasslibConfigWarning: The 'vary_rounds' option is deprecated as of Passlib 1.7, and will be removed in Passlib 2.0

Removing this code eliminates the deprecation warning, (plus, we didn't really want to vary the number of
rounds in the first place---we want precisely the number of rounds that we have configure).

PS. I wish github allowed me to put a review comment on the commit-message text itself. I often have more review comments for the commit messages than I do for the diff itself.